apis/validator/block.v3.yaml

```diff
@@ -83,36 +83,51 @@ get:
       content:
         application/json:
           schema:
-            title: ProduceBlockV3Response
-            type: object
-            required: [version, execution_payload_blinded, execution_payload_value, consensus_block_value, data]
-            properties:
-              version:
-                type: string
-                enum: [phase0, altair, bellatrix, capella, deneb, electra, fulu]
-                example: "electra"
-              execution_payload_blinded:
-                type: boolean
-                example: false
-              execution_payload_value:
-                type: string
-                example: "12345"
-              consensus_block_value:
-                type: string
-                example: "12345"
-              data:
-                anyOf:
-                 - $ref: "../../beacon-node-oapi.yaml#/components/schemas/Fulu.BlockContents"
-                 - $ref: "../../beacon-node-oapi.yaml#/components/schemas/Electra.BlockContents"
-                 - $ref: "../../beacon-node-oapi.yaml#/components/schemas/Electra.BlindedBeaconBlock"
-                 - $ref: "../../beacon-node-oapi.yaml#/components/schemas/Deneb.BlockContents"
-                 - $ref: "../../beacon-node-oapi.yaml#/components/schemas/Deneb.BlindedBeaconBlock"
-                 - $ref: "../../beacon-node-oapi.yaml#/components/schemas/Capella.BeaconBlock"
-                 - $ref: "../../beacon-node-oapi.yaml#/components/schemas/Capella.BlindedBeaconBlock"
-                 - $ref: "../../beacon-node-oapi.yaml#/components/schemas/Bellatrix.BeaconBlock"
-                 - $ref: "../../beacon-node-oapi.yaml#/components/schemas/Bellatrix.BlindedBeaconBlock"
-                 - $ref: "../../beacon-node-oapi.yaml#/components/schemas/Altair.BeaconBlock"
-                 - $ref: "../../beacon-node-oapi.yaml#/components/schemas/Phase0.BeaconBlock"
+            oneOf:
+              - title: "ProduceBlockV3Response (Gloas)"
+                type: object
+                required: [version, data, consensus_block_value]
```

Comment:
I removed the `execution_payload_blinded` and `execution_payload_value` from being required in the response post-gloas since there's no longer a concept of blinded blocks, and the block's "value" is now the bid, `BeaconBlockBody::SignedExecutionPayloadBid::value`

but lmk if these fields should be resurrected

not really sure if we need `consensus_block_value` at this point. it doesn't seem used in lighthouse, but I kept it as `required` since perhaps other clients use it

Reply:
introducing a `produceBlockV4` makes much mores sense imo

Reply:
produceBlockV4 definitely makes more sense